base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -263,7 +263,7 @@ BEGIN
     IDS_VERTICAL "Vertical"
     IDS_PRINTRES "%d x %d pixeli/cm"
     IDS_CANTPASTE "Nu a putut fi lipit din clipboard. Formatul de date este fie incorect, fie nesuportat."
-    IDS_SAVEERROR "Nu a putut fi salvată imaginea bitmap („hartă de biți”) în fișierul:\n\n%s"
-    IDS_CANTSENDMAIL "Nu a putut fi trimisă o scrisoare."
-    IDS_LOSECOLOR "Informațiile de culoare vor fi pierdute în această operațiune. Sunteți sigur că veți continua?"
+    IDS_SAVEERROR "Eșec în salvarea imaginii bitmap („hartă de biți”) în fișierul:\n\n%s"
```

Comment:
I (strongly) suggest you to avoid literal translations like "salvează". Use the adapted "păstrează" instead.